src/StatusQ/Components/StatusChatList.qml

```diff
@@ -29,199 +31,262 @@ Column {
+                                }
+                                dropArea.drag.source.chatListItem.newOrder = newOrder
+                                dropArea.drag.source.chatListItem.newCategoryId = statusChatListItem.categoryId
+                            }
+                        }
+                    }
+                }
+                Loader {
+                    id: draggedListItemLoader
+                    active: dragSensor.active
+                    sourceComponent: StatusChatListItem {
+                        property string chatListId: draggable.chatListItem.chatListId
+                        property var globalPosition: Utils.getAbsolutePosition(draggable)
+                        parent: QC.Overlay.overlay
+                        sensor.cursorShape: dragSensor.cursorShape
+                        Drag.active: dragSensor.active
+                        Drag.hotSpot.x: width / 2
+                        Drag.hotSpot.y: height / 2
+                        Drag.keys: ["chat-item"]
+                        Drag.source: draggable
+
+                        Component.onCompleted: {
+                            x = globalPosition.x
+                            y = globalPosition.y
                         }
+                        chatId: draggable.chatListItem.chatId
+                        categoryId: draggable.chatListItem.categoryId
+                        name: draggable.chatListItem.name
+                        type: draggable.chatListItem.type
+                        muted: draggable.chatListItem.muted
+                        dragged: true
+                        hasUnreadMessages: draggable.chatListItem.hasUnreadMessages
+                        hasMention: draggable.chatListItem.hasMention
+                        badge.value: draggable.chatListItem.badge.value
+                        selected: draggable.chatListItem.selected
+
+                        icon.color: draggable.chatListItem.icon.color
+                        image.isIdenticon: draggable.chatListItem.image.isIdenticon
+                        image.source: draggable.chatListItem.image.source
                     }
-                    onUnmute: statusChatList.chatItemUnmuted(model.chatId || model.id)
                 }
             }
+        }
+    }
 
-            DropArea {
-                id: dropArea
-                width: dragSensor.active ? 0 : parent.width
-                height: dragSensor.active ? 0 : parent.height
-                keys: ["chat-item-category-" + statusChatListItem.categoryId]
+    Loader {
+        id: popupMenuSlot
+        active: !!statusChatList.popupMenu
+    }
 
-                onEntered: reorderDelay.start()
-                onDropped: statusChatList.chatItemReordered(statusChatListItem.chatId, drag.source.originalOrder, statusChatListItem.DelegateModel.itemsIndex)
+    Loader {
+        id: emptyListDropAreaLoader
+        active: statusChatListItemsWrapper.implicitHeight == 0
```

Comment:
This doesn't seem to be enough. When the wrapper gets an `implicitHeight` at runtime, the loader is still active. Hence, we're updating `active` explicityly in `onImplicitHeightChanged`